content/docs/command-reference/exp/save.md

```diff
@@ -63,6 +63,10 @@ committing them to the Git repo. Unnecessary ones can be [cleared] with
   directory to be included in the experiment. This option can be used multiple
   times.
 
+- `-M <message>`, `--message <message>` - Custom commit message to use when
```

Comment:
Custom -> custom

Reply:
`commiting the experiment` 

It reads to me like it's about `dvc exp apply` or `dvc exp branch` even. Is it about custom refs? Should we just use `save` or `create` an experiment here?

Reply:
> Is it about custom refs? 

Not sure I follow the question, but yes, the actual git commit in the custom ref. 
When doing `exp branch` the branch will contain a commit with that message

Reply:
I mean, does add a message to a regular (non-commit) exp ref?

Reply:
> I mean, does add a message to a regular (non-commit) exp ref?

Yes

Reply:
yep, so, I would probably use some neutral language them "saves an experiment" vs "commits".